kyo-core/shared/src/main/scala/kyo/StreamCoreExtensions.scala

```diff
@@ -14,7 +14,10 @@ object StreamCoreExtensions:
                 case Absent =>
                     Loop.done
                 case Present(c) =>
-                    Emit.valueWith(c)(Loop.continue)
+                    if c.nonEmpty then
+                        Emit.valueWith(c)(Loop.continue)
+                    else
+                        Loop.continue
```

Comment:
@johnhungerford there was a flaky test due to an empty chunk sometimes being emitted. I've fixed it here, not sure if it's the best place. Failure: https://github.com/getkyo/kyo/actions/runs/15930299494/job/44937602854#step:6:7402